.github/workflows/docs-changelog.yaml

```
@@ -80,56 +80,93 @@ jobs:
+          ref: develop
 
-      - name: Checkout snippets
-        uses: actions/checkout@v4
-        with:
-          ref: ${{ inputs.reference }}
-          path: fa
-          sparse-checkout: |
-            changelog/snippets
+      - name: Determine next patch version
+        id: version
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2086:info:3:40: Double quote to prevent globbing and word splitting

(shellcheck)